linkerd/app/core/src/transport/labels.rs

```diff
@@ -74,6 +84,49 @@ impl FmtLabels for Key {
     }
 }
 
+impl ServerLabels {
+    fn inbound(
+        tls: tls::ConditionalServerTls,
+        target_addr: SocketAddr,
+        policy: PolicyLabels,
+    ) -> Self {
+        ServerLabels {
+            direction: Direction::In,
+            tls,
+            target_addr,
+            policy: Some(policy),
+        }
+    }
+
+    fn outbound(target_addr: SocketAddr) -> Self {
+        ServerLabels {
+            direction: Direction::Out,
+            tls: tls::ConditionalServerTls::None(tls::NoServerTls::Loopback),
+            target_addr,
+            policy: None,
+        }
+    }
+}
+
+impl FmtLabels for ServerLabels {
+    fn fmt_labels(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        self.direction.fmt_labels(f)?;
+        f.write_str(",peer=\"src\",")?;
+        (TargetAddr(self.target_addr), TlsAccept(&self.tls)).fmt_labels(f)?;
+
+        if let Some(policy) = self.policy.as_ref() {
+            for (k, v) in policy.server.iter() {
+                write!(f, ",srv_{}=\"{}\"", k, v)?;
+            }
+            for (k, v) in policy.authz.iter() {
+                write!(f, ",saz_{}=\"{}\"", k, v)?;
+            }
+        }
```

Comment:
nit/TIOLI: should this just be a `FmtLabels` impl for `PolicyLabels`, or do we expect them to show up elsewhere with different prefixes??

Reply:
the code for comma handling gets a bit more complicated than it's worth, imo